services-js/commissions-search/src/ResultList.js

```diff
@@ -1,128 +1,100 @@
-/* eslint-enable no-unused-vars */
 
-class ResultList extends Component {
-  makeSearchQuery() {
-    let getResults;
 
+/**
+ * This component will render the list of commissions based on the user's
+ * current search or filter parameters.
+ */
+class ResultList extends React.Component {
+  makeSearchQuery() {
     // need to construct query based on policy ids
     // if the facet array is empty then send null else send array of keys
-    if (
-      Object.keys(this.props.submittedAreas) === undefined ||
-      Object.keys(this.props.submittedAreas).length === 0
-    ) {
-      getResults = gql`{
-          commissions(query: "${
-            this.props.submittedKeywords
-          }", policyTypeIds: ${null}, hasOpenSeats: ${
-        this.props.submittedSeats === 'seats-open' ? true : null
-      }) {
-            id
-            name
-            policyType {
-              name
-            }
-            homepageUrl
-          }
-        }`;
-    } else {
-      getResults = gql`{
-          commissions(query: "${
-            this.props.submittedKeywords
-          }", policyTypeIds: [${Object.keys(
-        this.props.submittedAreas
-      )}], hasOpenSeats: ${
-        this.props.submittedSeats === 'seats-open' ? true : null
-      }) {
+    const submittedItems =
+      this.props.submittedAreas && Object.keys(this.props.submittedAreas).length > 0 ?
+        `[${Object.keys(this.props.submittedAreas)}]` : null
+    ;
+
+    return gql`{
+        commissions(query: 
+            "${this.props.submittedKeywords}",
+            policyTypeIds: ${submittedItems},
+            hasOpenSeats: ${this.props.submittedSeats === 'seats-open' ? true : null}
+        ) {
             id
             name
             policyType {
-              name
+                name
             }
             homepageUrl
-          }
-        }`;
-    }
-    return getResults;
+        }
+    }`;
   }
 
-  render() {
-    /* eslint-disable no-unused-vars */
-    function Result(props) {
-      // Check for null policyType
-      if (!props.policyName) {
-        return (
-          <li className="n-li">
-            <a
-              className="n-li-b n-li-b--r n-li-b--c n-li-b--fw n-li--in g g--mt0"
-              href={props.homepageUrl}
-            >
-              <div className="n-li-t g--8">{props.name}</div>
-              <div className="n-li-ty n-li-ty--r g--44" />
-            </a>
-          </li>
-        );
-      } else {
-        return (
-          <li className="n-li">
-            <a
-              className="n-li-b n-li-b--r n-li-b--c n-li-b--fw n-li--in g g--mt0"
-              href={props.homepageUrl}
-            >
-              <div className="n-li-t g--8">{props.name}</div>
-              <div className="n-li-ty n-li-ty--r g--44">
-                {props.policyName.name}
-              </div>
-            </a>
-          </li>
-        );
-      }
-    }
+  returnResult = ({ id, name, policyType, homepageUrl }) => {
```

Comment:
I think “return” in function names may be redundant. `renderCommissionItem` or `renderCommissionRow`?